### PR TITLE
upgrade some obsolete properties and methods

### PR DESCRIPTION
**Pull request by ZiyunShang**
Some class properties and methods is obsolete. They are deprecated in Revit 2019 and removed in the Revit 2020, so that some test cases can't run successfully.

### CODE REVIEW

File: src/Libraries/RevitNodes/RevitNodes.csproj [hunk 1]
```diff
@@ -57,11 +57,7 @@
     <Reference Include="ProtoGeometry">
       <HintPath>$(DYNAMOAPI)\ProtoGeometry.dll</HintPath>
       <Private>False</Private>
-    </Reference>
-    <Reference Include="RevitAPI">
-      <HintPath>$(REVITAPI)\RevitAPI.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
+    </Reference>    
     <Reference Include="System" />
```
Comment (philxia1) [inline src/Libraries/RevitNodes/RevitNodes.csproj]: why we change this project?

**Comment by philxia1** [comment]
@AndyDu1985 please review the api changed in this PR. Thanks.

File: src/Libraries/RevitNodes/RevitNodes.csproj (hunk 1 above)
Comment (ZiyunShang) [reply src/Libraries/RevitNodes/RevitNodes.csproj]: There is two RevitAPI.dll reference in the same csproj, I think it could be delete one.

Comment (QilongTang) [reply src/Libraries/RevitNodes/RevitNodes.csproj]: Sorry where is the other reference you mentioned? Which line in this file?

Comment (ZiyunShang) [reply src/Libraries/RevitNodes/RevitNodes.csproj]: @QilongTang line 49-52 is the same as the line 61-64, this is repeated.

**Pull request merged**